src/DssSpell.t.sol

```diff
@@ -138,32 +166,61 @@ contract DssSpellTest is DssSpellTestBase {
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // Insert new chainlog values tests here
-        // checkChainlogKey("CONTRACT_KEY");
-        // checkChainlogVersion("X.XX.X");
-
+        // RWA008
+        checkChainlogKey("RWA008");
+        checkChainlogKey("MCD_JOIN_RWA008_A");
+        checkChainlogKey("RWA008_A_URN");
+        checkChainlogKey("RWA008_A_OUTPUT_CONDUIT");
+        checkChainlogKey("RWA008_A_INPUT_CONDUIT");
+        checkChainlogKey("PIP_RWA008");
+
+        // RWA009
+        checkChainlogKey("RWA009_A_JAR");
+        checkChainlogKey("RWA009");
+        checkChainlogKey("MCD_JOIN_RWA009_A");
+        checkChainlogKey("RWA009_A_URN");
+        checkChainlogKey("RWA009_A_OUTPUT_CONDUIT");
+        checkChainlogKey("PIP_RWA009");
+
+        // RWA TOKEN FAB
+        checkChainlogKey("RWA_TOKEN_FAB");
+
+        checkChainlogVersion("1.13.3");
     }
 
-    function testNewIlkRegistryValues() private { // make public to use
+    function testNewIlkRegistryValues() public {
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // Insert new ilk registry values tests here
-        assertEq(reg.pos("TOKEN-X"), 49);
-        assertEq(reg.join("TOKEN-X"), addr.addr("MCD_JOIN_TOKEN_X"));
-        assertEq(reg.gem("TOKEN-X"), addr.addr("TOKEN"));
-        assertEq(reg.dec("TOKEN-X"), GemAbstract(addr.addr("TOKEN")).decimals());
-        assertEq(reg.class("TOKEN-X"), 1);
-        assertEq(reg.pip("TOKEN-X"), addr.addr("PIP_TOKEN"));
-        assertEq(reg.xlip("TOKEN-X"), addr.addr("MCD_CLIP_TOKEN_X"));
-        assertEq(reg.name("TOKEN-X"), "");
-        assertEq(reg.symbol("TOKEN-X"), "TOKEN");
+        // RWA008
+        (, address pipRwa008,,) = oracle.ilks("RWA008-A");
+
+        assertEq(reg.pos("RWA008-A"),    50);
+        assertEq(reg.join("RWA008-A"),   addr.addr("MCD_JOIN_RWA008_A"));
+        assertEq(reg.gem("RWA008-A"),    addr.addr("RWA008"));
+        assertEq(reg.dec("RWA008-A"),    DSTokenAbstract(addr.addr("RWA008")).decimals());
+        assertEq(reg.class("RWA008-A"),  3);
+        assertEq(reg.pip("RWA008-A"),    pipRwa008);
+        assertEq(reg.name("RWA008-A"),   "RWA008-A: SG Forge OFH");
+        assertEq(reg.symbol("RWA008-A"), "RWA008");
+
+        // RWA009
+        (, address pipRwa009,,) = oracle.ilks("RWA009-A");
+
+        assertEq(reg.pos("RWA009-A"),    51);
+        assertEq(reg.join("RWA009-A"),   addr.addr("MCD_JOIN_RWA009_A"));
+        assertEq(reg.gem("RWA009-A"),    addr.addr("RWA009"));
+        assertEq(reg.dec("RWA009-A"),    GemAbstract(addr.addr("RWA009")).decimals());
+        assertEq(reg.class("RWA009-A"),  3);
+        assertEq(reg.pip("RWA009-A"),    pipRwa009);
+        assertEq(reg.name("RWA009-A"),   "RWA009-A: H. V. Bank");
+        assertEq(reg.symbol("RWA009-A"), GemAbstract(addr.addr("RWA009")).symbol());
```

Comment:
```suggestion
        assertEq(reg.symbol("RWA009-A"), "RWA009");
```

Reply:
to keep consistency ^^

Reply:
Or maybe the other way around, I don't have a strong preference.

Reply:
I'm gonna keep the one with the call to `symbol()`, because that's what should match.

Reply:
sounds good

Reply:
Hmm I still do not see the change in the repo.

Reply:
Sorry, I didn't push right the way then, but it should be fixed now.